### DISCUSSION TITLE
Different data input for NICHES Vignette "Differential CellToCell Signaling Across Conditions"

### DISCUSSION THREAD

**Discussion by barbara-cortez** [msraredon/NICHES#13] [General]
Hi there,

I am trying to replicate the "Differential CellToCell Signaling Across Conditions" Vignette with a different dataset.
https://msraredon.github.io/NICHES/articles/04%20Differential%20CellToCell.html

My data it's also scRNAseq of two groups of PBMCs, stimulated and non-stimulated, but this data comes from GEO NCBI.
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE217868

I managed to download it in fasta format and to upload it to my bash terminal in RStudio, but I need to know how the format NICHES wants me to input my data in, so I can format it correctly.

My fasta file is a list of genes like this one:
@SRR22270117.11.2 11 length=150
CCAGCCTGGGCAATATGGCAAAAGCCTGTCCCTACAAAAAAAATAAAAAATCTTCTGTGGGTGGTGGTGTGTGTGTGATGG
TGTGTGCCTGTAGTCCCAGATACTTGGAAGGCTGAGGTGGGAGGATTGCTTGAGCCTGGGAGGTTGAGG

Any ideas on how I can format this in a NICHES friendly fashion? 

I could also use some insights on how to go through the other steps of the Vignette that uses Seurat data, such as in here:

table(ifnb@meta.data$stim,ifnb@meta.data$seurat_annotations)

I am imagining that if I replace the "ifnb"data with my own data, this command won't work, for my dataset, even if I have the Seurat package installed. Is that correct?

If so, could you give some ideas on how can I overcome this?

Thank you.

**Comment by msraredon** [comment]
I'm not entirely sure if I am understanding you correctly, but I think you may want to download the already aligned and extracted single-cell data from the same GEO reference (instead of the fasta files), which I think they have included as supplementary data here: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM6729183

You can then bring these files into R following a vignette like this one: https://satijalab.org/seurat/articles/pbmc3k_tutorial.html

And then once you are happy with the clustering and annotations, can run the NICHES vignette you mention or others. 

Does this help?

S

**Reply by barbara-cortez** [reply]
Hi Dr Raredon,

Yes, that was very helpful! Thank you very much.
